app/bundles/ReportBundle/Controller/Api/ReportApiController.php

```diff
@@ -51,7 +51,7 @@ public function __construct(CorePermissions $security, Translator $translator, E
      *
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function getReportAction(Request $request, $id)
+    public function getEntityAction(Request $request, $id)
```

Comment:
```suggestion
    public function getEntityAction(Request $request, int $id): Response
```

Reply:
We can not change signature of his method as Declaration must be compatible with [FetchCommonApiController->getEntityAction()](https://github.com/mautic/mautic/blob/dbfce0368858377035998c3c1acafa51b0815968/app/bundles/ApiBundle/Controller/FetchCommonApiController.php#L306)

Reply:
Thanks for pointing it.